src/formio/templates/radio.ejs

```diff
@@ -1,35 +1,32 @@
-<div class="{{ctx.ofPrefix}}form-choices">
+<!--This template is used by both the radio and the selectboxes components-->
+<div class="{{ctx.ofPrefix}}form-choices utrecht-form-fieldset__fieldset utrecht-form-fieldset__fieldset--html-fieldset">
   {% ctx.values.forEach(function(item) { %}
-    <div class="{{ctx.ofPrefix}}form-choices__choice utrecht-form-field utrecht-form-field--radio">
-        <div class="{{ctx.ofPrefix}}checkbox">
-              <{{ctx.input.type}}
-                ref="input"
+    <div class="{{ctx.ofPrefix}}form-choices__choice utrecht-form-field utrecht-form-field--{{ctx.input.attr['type']}}">
+      <input
+        ref="input"
+        class="utrecht-radio-button"
+        value="{{item.value}}"
+        id="{{ctx.id}}{{ctx.row}}-{{item.value}}"
 
-                {% for (var attr in ctx.input.attr) { %}
-                  {{attr}}="{{ctx.input.attr[attr]}}"
-                {% } %}
+        {% for (var attr in ctx.input.attr) { %}
+          {{attr}}="{{ctx.input.attr[attr]}}"
+        {% } %}
```

Comment:
How does `ctx.input.attr` get filled? No way of a `"class"` in there?

Reply:
It happens here: https://github.com/formio/formio.js/blob/v4.13.13/src/components/radio/Radio.js#L76 and then here: https://github.com/formio/formio.js/blob/v4.13.13/src/components/radio/Radio.js#L52
So yes, there is a class :S I will update it so that the class I added is added to the input instead of in the template